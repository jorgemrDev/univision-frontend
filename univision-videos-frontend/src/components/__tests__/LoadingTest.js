import React from "react";
import Loading from "../Loading";
import renderer from "react-test-renderer";

test("Render Loading correctly", () => {
  const component = renderer.create(<Loading></Loading>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
