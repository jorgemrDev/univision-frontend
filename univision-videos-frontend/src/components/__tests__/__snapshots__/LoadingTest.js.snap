// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Loading correctly 1`] = `
<div
  className="loading"
>
  <div
    className="ant-spin ant-spin-lg ant-spin-spinning"
  >
    <span
      className="ant-spin-dot ant-spin-dot-spin"
    >
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
    </span>
  </div>
</div>
`;
